src/blocks/stars.scss

```diff
@@ -0,0 +1,39 @@
+.card__rating {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &--margin--top {
+    margin-top: 16px;
+  }
+}
+
+.stars {
+  padding: 0;
+  display: flex;
+  gap: 4px;
+  list-style: none;
+
+  &__area {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+    background-size: auto;
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.card__reviews {
+  height: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 20px;
```

Comment:
The `line-height` is set to 20px, which is larger than the `height` of 16px for `.card__reviews`. This might cause vertical misalignment or overflow issues. Consider adjusting the `line-height` to better fit the `height`.